docs/platform/howto/project-support-center.rst

```diff
@@ -29,10 +74,70 @@ To create support tickets in the `Aiven Console <https://console.aiven.io/>`_:
 
 #. Click **Create ticket**. 
 
-The new support ticket is sent to our support team for review and you also receive a confirmation email. `Response times <https://aiven.io/support-services>`_ vary by case severity and support tier. 
+If you are on a paid support tier, your account team is also notified when you submit a ticket. 
+
+You can track the status of your tickets on the **My tickets** page. `Response times <https://aiven.io/support-services>`_ vary by case severity and support tier. 
+
+
+Add participants to a support ticket
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can CC other users in an open support ticket by adding their email addresses to the ticket. 
+
+* CCed emails that belong to Aiven users can also add comments and CC other emails on those tickets.
+* Non-users would only be able to reply to emails to add a comment? Can they CC others via email, adding them to the ticket?
+
+If you want to give every organization user access to all support tickets in your organization contact your account manager.
+
+To add someone to a support ticket:
+
+#. On the **My tickets** page, open the ticket.
+
+#. Click **Add to conversation**.
+
+#. Add the email addresses in the **CC** field separated by a space.
+
+#. Enter a comment and click **Submit**.
+
+
+Follow all support tickets in your organization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Super admin can get notifications for updates on all tickets in their organization. 
+
+#. Click **My tickets**.
+
+#. On the **Tickets in my organization** tab, click **Follow all tickets**.
+
+You will get email notifications for all updates on both existing and new tickets. You can unfollow them at any time.
+
+
+.. _create-har-files:
+
+Create HAR files
+~~~~~~~~~~~~~~~~~
+
+Aiven support may need information about the network requests that are generated in your browser when you experience a problem. Browsers can capture a log of these network requests in a HAR (HTTP Archive) file. 
+
+If the support team asks for a HAR file:
+
+#. Use your browser to create the HAR file while you go through the steps to reproduce the problem:
+
+   * Follow the `instructions for Internet Explorer/Edge, Firefox, and Chrome <https://toolbox.googleapps.com/apps/har_analyzer/>`_.
+   * For Safari, make sure you can access the `developer tools <https://support.apple.com/en-ie/guide/safari/sfri20948/mac>`_ and then follow the instructions for `exporting a HAR file <https://webkit.org/web-inspector/network-tab/>`_. 
+
+#. Replace sensitive data in the file with placeholders while retaining the JSON structure and format. Examples of sensitive data include:
+   
+   * Personal identifiers such as email addresses and phone numbers
+   * Authentication tokens or passwords
+   * Sensitive URLs
+   * Sensitive cookies or headers
+
```

Comment:
I think the #escalate in comments is missing, that this is the go-to option to vent when they are unhappy with the ticket handling.